### DISCUSSION TITLE
Azure Data Studio Theme

### DISCUSSION THREAD

**Discussion by vinodj** [microsoft/azuredatastudio#19569] [General]
I was wondering if anyone knows how to get the light theme with the blue borders, currently the default light theme has purple borders
Is it possible to change them?

See the attached screenshot.

![Cbp4x35zRy](https://user-images.githubusercontent.com/1154447/171058326-acc56aec-b222-4ba4-8f04-c1379d884309.png)

**Comment by pnr0318** [comment]
add new ADAS FOR THE AZURE

**Reply by vinodj** [reply]
I do not know what you mean. 
Please explain in a full sentence.

**Comment by erinstellato-ms** [comment]
@vinodj I'm not clear on where you see the purple border.  I only see blue for the light theme on my machine, and I don't see any purple in your screenshot.

**Comment by erinstellato-ms** [comment]
Hi @vinodj I am closing this discussion as I haven't heard back from you.  Feel free to reply if you are able to share more details (e.g. a screenshot of what you're seeing or would like to change).  Note that there are extensions in the library that offer alternative themes.
